### ISSUE TITLE
Anonymous Admin Bug

### ISSUE THREAD

**Issue by timea-techgirl** [telegramdesktop/tdesktop#10467]
### Steps to reproduce
1. Enable anonymous admin
2. Send a message in the group
3. Ask someone to quote your message and reply
4. You can see an arrow that shows the number of replies and a "View replies" option
5. Disable anonymous admin

### Expected behaviour
The arrow that indicates the number of replies should disappear after disabling anonymous admin

### Actual behaviour
The arrow that indicates the number of replies and the "View Replies" option on right click continue to show up even after disabling anonymous admin mode

### Configuration
**Operating system:**
Windows 10 x64

**Version of Telegram Desktop:**
2.6.1

**Used theme**:
Night

**Comment by RememberTheAir** [comment]
I'm missing something, why should it disappear?

**Comment by timea-techgirl** [comment]
Because it doesn't appear before enabling the anonymous admin option.

**Comment by john-preston** [comment]
@timea-techgirl The replies arrow should appear on any non-reply message with a reply (= any message, starting a thread).

**Comment by timea-techgirl** [comment]
@john-preston This arrow did not appear before enabling Anonymous admin. Right click and "View replies" opens a separate chat with replies.
![image](https://user-images.githubusercontent.com/7058205/110107483-f14da800-7dd0-11eb-9feb-a6aeb378be40.png)

Before enabling anonymous admin, there was no arrows or "View replies" option. The group was just like individual chats.
I created a new group just to test.

I don't wanna delete the group and create a new one. I hope this can be fixed. :(

**Comment by RememberTheAir** [comment]
Oh, maybe I get it. Changing an admin's permission triggered the conversion of the chat from normal group to supergroup. Have a read at [this article](https://telegra.ph/groups-08-26) to know more. Basically underneath Telegram uses two types of groups and not every feature is supported by both (like replies threads). There are some actions that trigger the conversion from one type (normal group, replies threads not supported) to the other (supergroup, replies threads supported): maybe this is what what happened in your chat. After changing that admin's permission, the chat was converted and started supporting replies threads (and thus you see the icon)

**Comment by timea-techgirl** [comment]
@RememberTheAir This is exactly what happened. It was upgraded to a supergroup after enabling the anonymous admin option.

It's a small group. Is there no way to go back to the old group type? Is creating a new group the only option?

**Comment by RememberTheAir** [comment]
> It's a small group. Is there no way to go back to the old group type? Is creating a new group the only option?

Yes, there is no way back and creating a new one is the only option (is it really necessary though?)

**Comment by timea-techgirl** [comment]
> > It's a small group. Is there no way to go back to the old group type? Is creating a new group the only option?
> 
> Yes, there is no way back and creating a new one is the only option (is it really necessary though?)

It's not necessary but I was used to the old group. This one feels bloated, more branches of chats for replied messages.. similar to a channel.

Does it affect chat history regardless of the group type?

**Comment by RememberTheAir** [comment]
> Does it affect chat history regardless of the group type?

Nope. The new features (such as the replies icon) will be available only for the chat history _after_ the group conversion to supergroup
